src/main/resources/liquibase/harness/change/expectedSnapshot/postgresql/changeCompositeTypeSchema.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this doesn't really check changeCompositeTypeSchema changetype. It could be that compositeType obj only has reference to schema by id, not it's name